### ISSUE TITLE
Capture a parameter

### ISSUE THREAD

**Issue by thoni56** [cgreen-devs/cgreen#265]
I often find that I want to get the value of a parameter and save that so I can use it later in the tests. But I cannot find a way to do that with the current set of features.

I propose a new constraint:

```
expect(fun, will_capture_parameter(parameter_name, local_variable));
```

**Comment by thoni56** [comment]
Branch https://github.com/cgreen-devs/cgreen/tree/capture_parameter contains one solution to this.

**Comment by thoni56** [comment]
This is now implemented and documented in 1.5.0
